[Privacy Gateway] New docs section (#8257)
cloudflare/cloudflare-docs: content/privacy-gateway/about.md

---
title: About
pcx_content_type: concept
weight: 2
---

# About

There are three main parties involved in Privacy Gateway:

- **Cloudflare**: Operators of the Privacy Gateway Proxy service.
- **Customers**: Companies and organizations that pay Cloudflare to proxy traffic to their application services through Privacy Gateway.
- **End users**: Users of customers’ applications.

---

## Cloudflare

### Data access

When a company or organization sets up Privacy Gateway, Cloudflare has access to the following data:

- Your device or computer’s IP address.

> - The application service you are using, including its DNS name and IP address.

Reviewer: ```suggestion
- The application service to which HTTP messages are relayed, including its DNS name and IP address.
```

- Metadata associated with the [request](/workers/runtime-apis/request/), including the type of browser, device operating system, hardware configuration, and timestamp of the request.

### Data limitations

Cloudflare cannot access the following data about an end user's request:

- Client application data, including contents of the encrypted application HTTP request.
- Target server application data, including contents of the encrypted application HTTP response.

### Data forwarding

> When requests go through Cloudflare's Privacy Gateway, Cloudflare relays the following information to the customer:

Reviewer: ```suggestion
As part of using Cloudflare's Privacy Gateway, Cloudflare will collect and report the following information to the customer:
```
